fades/main.py

```diff
@@ -260,6 +260,9 @@ def go(argv):
     # run forest run!!
     python_exe = 'ipython' if args.ipython else 'python'
     python_exe = os.path.join(venv_data['env_bin_path'], python_exe)
+    # add the virtualenv /bin path to the child PATH.
+    os.environ['PATH'] = "{}:{}".format(venv_data['env_bin_path'], os.environ.get('PATH'))
```

Comment:
If PATH is not in the environment, the `.get()` will return None, which will suck in the built path (it wil be `"....:None"`).

You could do `os.environ.get('PATH', "")`, but in the case of missing PATH you will end with a path that ends in the ":", would that be ok?

Otherwise you'd need to do something like: 
```
    environ_path = venv_data['env_bin_path']
    if 'PATH' in os.environ:
        environ_path += ':' + os.environ['PATH']
    os.environ['PATH'] = environ_path
```